### ISSUE TITLE
Regression when using ManifestStaticFilesStorage with 1.4

### ISSUE THREAD

**Issue by djw** [torchbox/wagtail#2369]
If using ManifestStaticFilesStorage as your STATICFILES_STORAGE, https://github.com/torchbox/wagtail/commit/824a7c50e272c3f1492551f015716a771f8cb639 introduces a requirement that your manifest file is present when loading your app.

This can be verified by putting

```
STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
```

into wagtail/tests/settings.py, and running the tests on 1.3 and 1.4.

I run celery in an asset-less container, which now throws exceptions on startup when using Wagtail 1.4.

**Comment by kaedroho** [comment]
Thanks for reporting this @djw

**Comment by m1kola** [comment]
Fixed in #2404 by @kaedroho and will be included in v1.4.2.

Thanks for reporting @djw!
